cmake/FindSDL3.cmake

```diff
@@ -0,0 +1,7 @@
+include(FetchContent)
```

Comment:
May I suggest to add a status message here?
```
message(STATUS "Fetching SDL2 files from ${SDL_GIT}")
```

Reply:
Sorry, I put SDL2, but the actual version is defined by the GIT_TAG, so I guess it would be nice to use that in the message :)